src/functions/math/pow.js

```diff
@@ -0,0 +1,15 @@
+module.exports = d => {
+    const data = d.util.aoiFunc(d);
+    if (data.err) return d.error(data.err);
+
+    const [number, power] = data.inside.splits;
+
+    if (isNaN(number) || number.trim() === '') return d.aoiError.fnError(d, 'custom', {inside: data.inside}, 'Invalid Number Provided In');
+    if (isNaN(power) || power.trim() === '') return d.aoiError.fnError(d, 'custom', {inside: data.inside}, 'Invalid Power Provided In');
+
+    data.result = Math.pow(Number(number), Number(power));
```

Comment:
Add chaining options so users can chain them like $pow[1;2;3]
so we get Math.pow(Math.pow(1,2),3)